### PR TITLE
rbac's generated file's EOL-characters fixed #18383

### LINKED ISSUE #18383: RBAC's PhpManager generates PSR12 non-compliant file
### What steps will reproduce the problem?

Run PHP_Codesniffer with the PSR-12 ruleset against file generated by `yii\rbac\PhpManager`:

`php vendor/squizlabs/php_codesniffer/bin/phpcs --standard=PSR12 ./app/config/rbac/items.php`

### What is the expected result?

No errors or warnings

### What do you get instead?

```
FILE: ...\app\config\rbac\items.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 1 LINE
----------------------------------------------------------------------
 1 | ERROR | [x] Header blocks must be separated by a single blank
   |       |     line
 1 | ERROR | [x] End of line character is invalid; expected "\n" but
   |       |     found "\r\n"
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 16.63 secs; Memory: 28MB
```

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.36
| PHP version      | 7.4.6
| Operating system | Microsoft Windows 10 Pro 10.0.19042 N/A Build 19042

### PR DESCRIPTION
**Pull request by perlexed**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #18383

### CODE REVIEW

**Review by githubjeka — APPROVED** [review]

File: framework/rbac/PhpManager.php [hunk 1]
```diff
@@ -796,7 +796,7 @@ protected function loadFromFile($file)
      */
     protected function saveToFile($data, $file)
     {
-        file_put_contents($file, "<?php\nreturn " . VarDumper::export($data) . ";\n", LOCK_EX);
+        file_put_contents($file, "<?php\r\n\r\nreturn " . VarDumper::export($data) . ";\r\n", LOCK_EX);
```
Comment (samdark) [inline framework/rbac/PhpManager.php:799]: How about using \n without \r?

Comment (perlexed) [reply framework/rbac/PhpManager.php:799]: now that is look at it it makes sense... should I make another pull-request?

Comment (samdark) [reply framework/rbac/PhpManager.php:799]: No. Just push another commit to the same branch.

**Comment by samdark** [comment]
Merged.

**Pull request merged**